custom_components/myskoda/binary_sensor.py

```diff
@@ -325,6 +325,35 @@ def is_on(self):  # noqa: D102
         return self.vehicle.status.detail.bonnet == OpenState.OPEN
 
 
+class SunroofOpen(MySkodaBinarySensor):
+    """Detectes whether the sunroof is open."""
+
+    def __init__(self, coordinator: DataUpdateCoordinator, vehicle: Vehicle) -> None:  # noqa: D107
```

Comment:
nit/later: if we're going to disable D107 for all/most init methods we might as well disable it globally in pyproject.toml

```
[tool.ruff.lint]
ignore = [
    "D171"   # undocumented-public-init
]
```

Reply:
Agree